cairo-m/Cargo.toml

```diff
@@ -16,3 +16,12 @@ stwo-prover = { git = "https://github.com/starkware-libs/stwo", features = [
   "parallel",
 ], rev = "8b5436d" }
 uniffi = { version = "0.29.3", features = ["cli"] }
+
+
+[profile.release]
+opt-level = 3
+lto = true
+codegen-units = 1
```

Comment:
> No LTO is performed if codegen units is 1
https://doc.rust-lang.org/cargo/reference/profiles.html#lto

Don't we want to put true,"fat" or "thin"